### PR TITLE
testbed/arcyd/docker-test.sh: redo as testshell

### PR DESCRIPTION
**Pull request by aevri**
Replace 'sleep 5' in docker-test with an invocation of bash, so that
users can easily play with a running instance.

Rename to 'docker-testshell'.

Provide documentation for some common things to try out in the
test shell.

Test Plan:
```
$ ./precommit.sh
$ ./testbed/arcyd/docker-testshell.sh
    .. try seeing the logs ..
    .. try seeing the info log ..
    .. try standing up a new instance ..
    $ exit
```

### CODE REVIEW

File: testbed/arcyd/docker-testshell.sh [hunk 1]
```diff
@@ -64,7 +64,29 @@ docker exec arcydtest-arcyd arcyd-do add-repo phabweb mygit git://arcydtest-git/
...
+$ docker exec arcydtest-arcyd cat /var/arcyd/var/log/info
+
+Start a new 'arcyd2' container (to clean up yourself):
+$ docker run -d --link phab-web --name arcyd2 arcyd git://arcydtest-git/arcyd
+
+EOF
+set +e
+bash
+set -x
+
```
Comment (hashpling) [inline testbed/arcyd/docker-testshell.sh]: Consider turning `set -e` back on for consistency?

Comment (aevri) [inline testbed/arcyd/docker-testshell.sh]: Yup, I agonised about that too much before and after PR :)

There's an unrelated commit I should make to improve the post-bash cleanup.

Will `set -e` for this change. Thanks!

**Comment by aevri** [comment]
Thanks!
